test/sqllogictest/webhook.slt

```diff
@@ -12,9 +12,13 @@ mode cockroach
 # Start from a pristine server
 reset-server
 
+# Create a cluster to install all of our webhook sources on.
```

Comment:
Can you add a test asserting that creating a webhook source specifying SIZE fails?